hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/testutils/HoodieClientTestHarness.java

```diff
@@ -489,12 +490,20 @@ public SparkRDDReadClient getHoodieReadClient(String basePath) {
   }
 
   public SparkRDDWriteClient getHoodieWriteClient(HoodieWriteConfig cfg) {
+    return getHoodieWriteClient(cfg, false);
+  }
+
+  public SparkRDDWriteClient getHoodieWriteClient(HoodieWriteConfig cfg, Boolean makeClusteringBroken) {
     if (null != writeClient) {
       writeClient.close();
       writeClient = null;
     }
-    writeClient = new SparkRDDWriteClient(context, cfg);
-    return writeClient;
+    if (makeClusteringBroken) {
```

Comment:
can we not introduce makeClusteringBroken to this test base. I don't see many usages of this for now. so, lets keep it within the test class only.